.github/workflows/ci.yml

```diff
@@ -14,18 +14,41 @@ env:
   try-runtime-uri: wss://eden-rpc.dwellir.com:443
 
 jobs:
-  test:
+  clippy:
+    runs-on: ubuntu-latest-8-cores
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions-rust-lang/setup-rust-toolchain@v1
+        with:
+          toolchain: ${{ env.nightly }}
+          target: ${{ env.target }}
+          components: clippy
+      - name: Clippy
+        run: |
+           unset RUSTFLAGS
+           cargo clippy --all-features -- -W clippy::type_complexity -W clippy::identity_op -W clippy::boxed_local -D dead_code
```

Comment:
`cargo clippy` did not work as well as the old addon. (warnings are not visible on the code review screen) Will look for a maintained fork.